src/main/java/uk/gov/companieshouse/company/profile/controller/CompanyProfileController.java

```diff
@@ -78,4 +79,20 @@ public ResponseEntity<Void> addExemptionsLink(
         companyProfileService.addExemptionsLink(contextId, companyNumber);
         return ResponseEntity.status(HttpStatus.OK).build();
     }
+
+    /**
+     * Delete a company exemptions link on a company profile for the given company number.
+     *
+     * @param companyNumber The number of the company
+     * @return no response
+     */
+    @DeleteMapping("/company/{company_number}/links/exemptions")
```

Comment:
This should be a PATCH as we are removing part of a record rather than deleting a record. Uri will need to become "/company/{company_number}/links/exemptions/delete" to differentiate between PATCH for addExemptionsLink